modules/standard/List.chpl

```diff
@@ -186,6 +186,13 @@ record list {
     length = 0;
   }
 
+  /*
+    Destructor
+   */
+  proc ~list(){
```

Comment:
We've recently started replacing destructors like this with deinit(), so if you could change this over to:

````chapel
proc deinit() {
````